### ISSUE TITLE
CI is failing

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-spec#867]
It looks like there is a problem in some of the dependencies needing an update.

Below is the output from the failed run:

```
        File "/tmp/pip-build-env-cdrc14e2/overlay/lib/python3.12/site-packages/setuptools/_distutils/command/sdist.py", line 251, in add_defaults
          self._add_defaults_ext()
        File "/tmp/pip-build-env-cdrc14e2/overlay/lib/python3.12/site-packages/setuptools/_distutils/command/sdist.py", line 336, in _add_defaults_ext
          self.filelist.extend(build_ext.get_source_files())
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "<string>", line 201, in get_source_files
        File "/tmp/pip-build-env-cdrc14e2/overlay/lib/python3.12/site-packages/setuptools/_distutils/cmd.py", line 107, in __getattr__
          raise AttributeError(attr)
      AttributeError: cython_sources
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

Notice:  A new release of pip is available: 23.2.1 -> 23.3.1
Notice:  To update, run: pip install --upgrade pip
Error: Process completed with exit code 1.
```

Complete CI output: 
[errors.txt](https://github.com/spdx/spdx-spec/files/13230044/errors.txt)

**Comment by puneeth072003** [comment]
I'd be more than happy to take up this issue

**Comment by goneall** [comment]
@licquia - can you take a look?

**Comment by licquia** [comment]
Ref: https://github.com/yaml/pyyaml/pull/702

The odd thing is that there's no reason why we shouldn't always be building this test with newer PyYAML, which has a workaround for this problem.  I'll try pushing an extra requirement on newer PyYAML to see if that fixes the problem.

**Comment by licquia** [comment]
Tried pulling that fix into the 2.3.1 build, but it doesn't work there.  One of the requirements there requires PyYAML 5, and so the requirements are uninstallable if we insist on 6.

I've pushed a new workaround on the 2.3.1 branch which looks like it works on my system.

**Comment by kestewart** [comment]
CI is no longer failing.
